amf-aml/shared/src/main/scala/amf/aml/internal/semantic/SemanticExtensionsFacade.scala

```diff
@@ -10,39 +11,47 @@ import amf.core.internal.plugins.render.RenderConfiguration
 import amf.core.internal.render.SpecOrdering
 import org.yaml.model.YMapEntry
 
-class SemanticExtensionsFacade private (val registry: AMLRegistry) {
+class SemanticExtensionsFacade private (annotation: String, val registry: AMLRegistry) {
```

Comment:
Will we have one extension facade for semex ? If so, why ?

Reply:
I need to pre process the registry reducing all the extensions presents for those who come from the aliased reference. 
Also, need to strip the alias from the key at web api before invoke the AML abstraction)